### PR TITLE
Add a bunch of new apps and update old links

### PR DESCRIPTION
**Pull request by DimitrisPa**

### CODE REVIEW

**Comment by DimitrisPa** [comment]
### Complete List
**Audio**
- Cider
- EasyEffects
- Mousai
- Soundux

**3rd party Client**

- ModernDeck

**Chat Client Utilities**

- BetterDiscord

**Official Client**

- Mattermost
- Revolt
- Rocket.chat
- Session

**Data Backup and Recovery**

- Pika Backup

**Supporting Tools**

- Czkawka

**E-Book Utilities**

- Foliate

**Editors**

- LunarVim

**Gaming Applications**

- Bottles
- GameHub
- GameMode
- GOverlay
- Heroic Games Launcher
- MultiMC
- PolyMC
- ProtonUp-Qt

**Machine Emulators**

- Citra
- PCSX2
- RPCS3
- Ryujinx
- yuzu

**Browser**

- LibreWolf
- ungoogled-chromium

**Web Service Client**

- FreeTube

**Note Taking**

- Logseq
- Notepad Next
- Standard Notes
- Turtl

**Time and Task**

- Kalendar
- Solanum

**Password Manager**

- gopass
- Psono
- VaultWarden

**Disk Utilities**

- popsicle
- Ventoy

**Other**

- Flatseal

**Video**

- Celluloid
- Stremio

**Command Line Utilities**

- bat
- btop
- duf
- fd
- yt-dlp

File: README.md
Comment (luong-komorebi) [inline README.md:509]: interesting

File: README.md
Comment (luong-komorebi) [inline README.md:718]: Hopefully one day they will put linux on their download screen

**Review by luong-komorebi — DISMISSED** [review]
I cant thank you enough!

This has been one of the most valuable and enormous contribution that I have ever received. 
To be honest, I kinda lost it til the Markdown section because there are so many apps and believe  it not more than half of them I installed on my machine and tried it myself, which is tedious, but it is fine since I am a nerd. I really love how all of your suggestions comes with validity or at least some kinds of pages that do simple landing page to normal users, and the apps are actually practical.

Since I have applied some trust on you already, after this is merged, you are more than welcomed to become a maintainer of this list if you wish to.

Let me know when this should be merged (seems like you are still working on it).  Again, thank you.

**Comment by DimitrisPa** [comment]
Thank you very much for your kind words, I am glad that i helped you. Thanks for the offer regarding the maintenance role! I would be more than happy to help. Also, I have found more application (i.e Nemiver,llpp) that they are officially Discontinued/Archived and I believe there are better/modern alternatives for these and should be removed from the list if that's ok with you of course. I can create a new PR in order to remove them.

**Comment by luong-komorebi** [comment]
> should be removed from the list

It can be done in this PR 👍🏻

**Comment by DimitrisPa** [comment]
### Removed from the list ###

**Audio**

- Gradio (Archived) 

**Desktop Themes**

- Adapta Theme (Discontinued)
- EvoPop Theme (Discontinued last update 2017)
- Flatabulous (Discontinued last update 2016)
- Arc-Flatabulous Theme (Discontinued last update 2018)
- Numix gtk (Discontinued last update 2017)

**Supporting Tools**

- Nemiver (Archived) 

**E-Book Utilities**

- llpp(From Arch Wiki : The github repository has all the files were deleted and replaced.)

**Email**

- wmail (Discontinued last update 2017)

**Miscellaneous**

- Ramme (Discontinued last update 2018)

**Web Service Client**

- FeedReader (Discontinued)

**Proxy**

- electron-ssr (Discontinued)

**Password Manager**

- Encryptr (Discontinued)

**Terminal**

- termite (Discontinued)
- upterm (Discontinued)

**Review by luong-komorebi — DISMISSED** [review]

**Pull request merged**
